cli/src/pcluster/validators/cluster_validators.py

```diff
@@ -794,13 +795,24 @@ class HeadNodeImdsValidator(Validator):
     def _validate(self, imds_secured: bool, scheduler: str):
         if scheduler is None:
             self._add_failure(
-                f"Cannot validate IMDS configuration with scheduler {scheduler}.",
+                f"Cannot validate IMDS configuration with scheduler {scheduler}."
+                f"Valid values are: {SUPPORTED_SCHEDULERS}.",
+                FailureLevel.ERROR,
+            )
+        elif imds_secured is None:
+            self._add_failure(
+                f"Cannot validate IMDS configuration with Imds.Secured {imds_secured}."
```

Comment:
minor: this is going to result in an error message claiming that the value of this parameter was set to `None`, when more likely it means there's a bug in our handling of the config file. i can't think of a better error message at the moment, but i could see that as being confusing.

Reply:
I agree with you about the confusion between customer misconfig and bug. 
In any case, I would prefer to highlight a bug with an error message rather than assuming it does not happen and forward wrong values. 
In this way, if the bug is there, a customer will see the message repeatedly and at the end will submit a clear support request.

Reply:
The basic idea is that Validators should provide "semantic" error messages, without mentioning configuration parameters.

In this case we can print something like: "Cannot validate IMDS configuration if scheduler is not set" or "if IMDS Secured is not set".

Reply:
So, shouldn't we provide a hint to the customer about valid values?

Reply:
Talked with Enrico offline. I finally agree on that.